src/Menu/Provider/GroupMenuProvider.php

```diff
@@ -172,7 +173,11 @@ private function generateMenuItem(array $item, array $group): ItemInterface
         if (isset($item['admin']) && !empty($item['admin'])) {
             $admin = $this->pool->getInstance($item['admin']);
 
-            $options = $admin->generateMenuUrl('list', [], $item['route_absolute']);
+            $options = $admin->generateMenuUrl(
+                'list',
+                [],
+                $item['route_absolute'] ? UrlGeneratorInterface::ABSOLUTE_URL : UrlGeneratorInterface::ABSOLUTE_PATH
```

Comment:
I would prefer an other way to support all possible path values: https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Routing/Generator/UrlGeneratorInterface.php#L37-L55

```suggestion
                in_array($item['route_absolute'], [/* possible values*/]) ? $item['route_absolute'] : UrlGeneratorInterface::ABSOLUTE_PATH
```

Reply:
I agree, but this reference is intended to be used with [`RoutingExtension::buildOptions()`](https://github.com/KnpLabs/KnpMenu/blob/b6aade272c345b6fbd07fce5929a761cba0909b8/src/Knp/Menu/Integration/Symfony/RoutingExtension.php#L21-L36) at "knplabs/knp-menu", which does not distinguish other values than `true` or `false`.
I guess this obbeys to [how Symfony handled these references in the past](
https://github.com/symfony/symfony/blob/6a9c223b066a2553b8f59fd26f84ea630eff403c/src/Symfony/Component/Routing/Generator/UrlGeneratorInterface.php#L39-L56).
IMO, the only useful mapping here is the absolute URL by one side (`true`) and the rest of possible values otherwise (`false`, representing relative domain, relative scheme, and relative path).
Even if we provide a way in our configuration for using the 4 possible values, there will be no chance to make "knplabs/knp-menu" respect them (at least in the current version).